pkg/alertmanager/state_persister.go

```diff
@@ -85,6 +99,8 @@ func (s *statePersister) iteration(ctx context.Context) error {
 }
 
 func (s *statePersister) persist(ctx context.Context) error {
+	s.persistTotal.Inc()
```

Comment:
If you track it before `s.state.Position() != 0` then it's value doesn't match with `s.persistFailed`  because only the replica at position 0 can fail (think about computing the % of failures). I would track this metric after the check `s.state.Position() != 0`.

Reply:
Agreed, not sure why I put it here.... Probably sleep deprivation!